src/components/DocumentDescription/messages.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+    tutelaTitle: "¿Qué es una tutela?",
+    tutelaDescription: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+    contratoTitle: "¿Qué es un contrato?",
+    contratoDescription: "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.",
```

Comment:
What about adding another key layer that represents the type of document instead of listing all possible keys of every type of document?
`{
  tutela: { title, description },
  contrato: { title, description }
}`
I think this structure is better cause we could start structuring (maybe even use a ts type) to tell what information is inside each document. For now, we can mark this object as a 
`
const type documents = 'tutela | contrato'
interface documentContent = {
    title: string;
    description: string;
}
Record<documents, documentContent>
`